mpa/modules/models/segmentors/class_incr_segmentor.py

```diff
@@ -45,9 +45,8 @@ def load_state_dict_pre_hook(model, model_classes, chkpt_classes, chkpt_dict, pr
 
         model_dict = model.state_dict()
         param_names = [
-            'decode_head.0.conv_seg.weight',
-            'decode_head.0.conv_seg.bias',
-            'decode_head.1.conv_seg.weight',
+            'decode_head.conv_seg.weight',
+            'decode_head.conv_seg.bias',
```

Comment:
Is this enough for arch change?
Can we load old OTE model weights from, for example, SC1.1 model with OCR head?

Reply:
Currently it does not support. Let me update it on next PR.